app/src/main/res/values-de/strings.xml

```diff
@@ -267,7 +267,7 @@
     <string name="importBookmarksFileTitle">Wählen Sie aus, welche Lesezeichendatei Sie importieren möchten</string>
     <string name="importBookmarksError">Etwas ist schief gegangen und es konnten keine Lesezeichen importiert werden</string>
     <string name="importBookmarksEmpty">Diese Datei enthält keine Lesezeichen, die importiert werden können</string>
-    <string name="importBookmarksSuccess">%1$d Lesezeichen wurden erfolgreich importiert</string>
+    <string name="importBookmarksSuccess" instruction="Placeholder is a number">%1$d Lesezeichen wurden erfolgreich importiert</string>
```

Comment:
nit: you're more specific about "a number of what" in other places. doubt it's needed though to say "placeholder is a number of bookmarks imported" ?

Reply:
For numbers is kinda obvious and it is more important when it comes to strings. In any case there is a project to do this consistently and with more correct instructions.